myTSReduxApp/store/pageReducer/hook.ts

```diff
@@ -0,0 +1,5 @@
+import {useSelector} from 'react-redux';
+
+import {selectPageList} from './select';
+
+export const usePageList = () => useSelector(selectPageList);
```

Comment:
I particularly like the idea of providing a `usePageList` hook that in turns encapsulates state using the `selectPageList` selector.

As the article states, it's good not to expose the structure of state to its consumers, with a hook+selector, you can hide those details and if you restructure the state later on, you don't impact the hooks' users.